src/azure/cli/commands/parameters.py

```diff
@@ -65,6 +65,9 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
         return list(enum_type.__dict__[ENUM_MAPPING_KEY].keys())
     return completer
 
+def get_enum_choices(enum_type):
+    return [x.value for x in enum_type]
```

Comment:
How is this different from the above 'get_enum_type_completion_list'? They seem very similar.